src/components/CommentSection/CommentSectionContainer.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import './commentsection.css';
+import CommentLoad from './CommentLoad';
+
+const CommentSectionContainer = props => {
+    return(
+        <div className={"commentSectionContainer"}>
+
+            {props.comments.map((username, comment) => {
```

Comment:
The second parameter of .Map is for index of the current iteration. This will probably get you into some problems. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map